lib/Connector/Sabre/LockPlugin.php

```diff
@@ -118,6 +118,9 @@ public function checkLock(RequestInterface $request): void {
 
 		// Throw an error, if the user-agent does not support end to end encryption
 		$userAgent = $request->getHeader('user-agent');
+		if ($userAgent === '') {
+			$userAgent = '--';
```

Comment:
Thanks :+1: 

What's wrong with an empty string? ;)

Reply:
- Maintaining the same throw path a few lines down w/o needing to customize the explanation just for this 
- Consistency with how we log an empty User-Agent elsewhere :man_shrugging: